### ISSUE TITLE
netconvert VISSIM: changes in VISSIM .inpx file structure

### ISSUE THREAD

**Issue by m-kro** [eclipse/sumo#5915]
A student just pointed me to this issue: With the current VISSIM v.11, networks use the following tags for road geometries in the VISSIM XML .inpx format:

- linkPolyPoint (was: point3D)
- linkPolyPts (was: points3D)

This can be triggered by the network version in the root tag:
`<network version="603" vissimVersion="11.00 - 09 [78629]">`

**Comment by namdre** [comment]
May this sample be used in the public test suite?

**Comment by m-kro** [comment]
Yes for sure. But maybe you wait a few hours and I go back to create a more universal example with more features (like traffic lights).

**Comment by m-kro** [comment]
Here is a little more advanced small VISSIM network with traffic lights. Even if I think those are not read at the moment for inpx files, right?
[VISSIM_BS_Bueltenweg.zip](https://github.com/eclipse/sumo/files/3505461/VISSIM_BS_Bueltenweg.zip)

**Comment by namdre** [comment]
- at the least the import doesn't crash and creates a somewhat reasonable network
- overlapping junctions that should interact but are topologically distinct won't work well in sumo. Not sure how much of this is vissim style and how much is mishandling by netconvert (junctions 3 and 5)
- signalcontrollers are not currently imported. Is the .sig format vissim specific or is this a more general format?

**Comment by m-kro** [comment]
I presume .sig files are specific. The signal program editor VISSIG reads/writes this format and hands it over to VISSIM. Editing of signal plans cannot be done in the proper VISSIM GUI.
